modules/local/summarizeBBMerge/main.nf

```diff
@@ -0,0 +1,17 @@
+process SUMMARIZE_BBMERGE {
+    label "base"
+    label "single"
+    input:
+        tuple val(sample), path(merged)
+    output:
+        path("${sample}_bbmerge_merged_summary.txt")
+    script:
+        """
+        zcat ${merged} | awk '
```

Comment:
Can we also get the post-merge sequence? This isn't currently included in the HV table and would be good to have.

(Ditto the post-concat sequence for unmerged read pairs, though this is somewhat easier to derive from the paired sequences.)

Reply:
Also, what's your plan for the merge status?

Reply:
@willbradshaw chatted about this during out meeting, but this was in collapseHV, which has now been moved to it's own module.